redbot/core/core_commands.py

```diff
@@ -1919,7 +1919,12 @@ async def stream(self, ctx: commands.Context, streamer=None, *, stream_title=Non
     async def _username(self, ctx: commands.Context, *, username: str):
         """Sets [botname]'s username."""
         try:
-            await self._name(name=username)
+            if len(username) > 32:
+                await ctx.send(_("Failed to change name. Must be 32 characters or fewer."))
+                return
+            else:
+                print("USERNAME LENGTH OK")
```

Comment:
We don't use `print` statements for this. So this should either be removed or changed to `log.debug` if you want to keep it although I don't really see a need for it beyond testing the function worked.

Reply:
Woops good catch, forgot to remove it